backend/leanstore/storage/btree/core/BTreeNode.cpp

```diff
@@ -142,52 +142,42 @@ u32 BTreeNode::spaceUsedBySlot(u16 s_i)
 // -------------------------------------------------------------------------------------
 // right survives, this gets reclaimed
 // left(this) into right
-bool BTreeNode::merge(u16 slotId, ExclusivePageGuard<BTreeNode>& parent, ExclusivePageGuard<BTreeNode>& right)
+bool BTreeNode::merge(ExclusivePageGuard<BTreeNode>& self, u16 slotId, ExclusivePageGuard<BTreeNode>& parent, ExclusivePageGuard<BTreeNode>& right)
```

Comment:
Was it buggy or why you made this change?

Reply:
I need access to both (left and right) bfs.header.tracker (To merge the two Timestamp-Trackers)